### PR TITLE
Update steps on "How to collect Logs"

### PR DESCRIPTION
**Pull request by anurag1128**
Collect and enable logging
It’s important to collect logs as soon as an issue occurs. The log can be collected together with just a couple clicks.
Windows
Right-click on the Teams icon in the system tray and choosing Collect support files. 
Mac
Select the Help menu and choose Collect support files.
Debug, Desktop and Media logs will be collected in one folder with the name MSTeams Diagnostics Log <local data and time>. This folder can be compressed and shared when you open a support request with Microsoft Support. This folder will contain folders for Desktop, Meeting (Media) and Debug (web). You can collect the files using the keyboard shortcuts -
Windows
Crtl + Alt + Shift + 1
Mac
Option + Command + Shift + 1
Media logging is turned off by default. To enable Media logging, users must turn on the option in the Teams client. Go to Settings > General, select the Enable logging for meeting diagnostics (requires restarting Teams). The Teams client must be restarted for logging to begin.
Note If Media logging is enabled there will be additional files included in the Meeting folder which are necessary for investigating audio and video issues. If Media logging is not enabled there will be a limited number of logs available.
Note In this article, the term Debug logs refers to the logs that are used for troubleshooting. However, the files that are generated for these logs will contain the term diagnostic logs in their names.
The following table outlines the various clients and their associated logs. 
TABLE 1
Client	Debug	Desktop	Media
Web	X	-	-
Windows	X	X	X
Mac OSX	X	X	X
Linux	X	X	X
iOS	-	-	-
Android	-	-	-
For a complete list of supported operating systems and browsers, see Get clients for Microsoft Teams.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/log-files.md [hunk 1]
```diff
@@ -29,6 +29,35 @@ There are three types of log files automatically produced by the client, which c
 
 -   Desktop logs
 
+**Collect and enable logging**
+=================================================
+
+It’s important to collect logs as soon as an issue occurs. The log can be collected together with just a couple clicks.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:35]:
```suggestion
It’s important to collect logs as soon as an issue occurs. The log can be collected together with just a couple of clicks.
```

File: Teams/log-files.md [hunk 2]
```diff
@@ -29,6 +29,35 @@ There are three types of log files automatically produced by the client, which c
...
+Windows
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:36]:
```suggestion
Windows:

```

File: Teams/log-files.md [hunk 3]
```diff
@@ -29,6 +29,35 @@ There are three types of log files automatically produced by the client, which c
...
+Right-click on the Teams icon in the system tray and choosing Collect support files.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:37]:
```suggestion
Right-click on the Teams icon in the system tray and choose **Collect support files**. 
```

File: Teams/log-files.md [hunk 4]
```diff
@@ -29,6 +29,35 @@ There are three types of log files automatically produced by the client, which c
 
 -   Desktop logs
 
+**Collect and enable logging**
+=================================================
+
+It’s important to collect logs as soon as an issue occurs. The log can be collected together with just a couple clicks.
+Windows
+Right-click on the Teams icon in the system tray and choosing Collect support files. 
+Mac
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:38]:
```suggestion
Mac:

```

File: Teams/log-files.md [hunk 5]
```diff
@@ -29,6 +29,35 @@ There are three types of log files automatically produced by the client, which c
...
+Select the Help menu and choose Collect support files.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:39]:
```suggestion
Select the Help menu and choose **Collect support files**.

```

File: Teams/log-files.md [hunk 6]
```diff
@@ -29,6 +29,35 @@ There are three types of log files automatically produced by the client, which c
...
+Debug, Desktop and Media logs will be collected in one folder with the name MSTeams Diagnostics Log <local data and time>. This folder can be compressed and shared when you open a support request with Microsoft Support. This folder will contain folders for Desktop, Meeting (Media) and Debug (web). You can collect the files using the keyboard shortcuts -
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:40]:
```suggestion
Debug, Desktop, and Media logs will be collected in one folder with the name MSTeams Diagnostics Log <local data and time>. This folder can be compressed and shared when you open a support request with Microsoft Support. This folder will contain folders for Desktop, Meeting (Media), and Debug (web). You can collect the files using the following keyboard shortcuts:

```

File: Teams/log-files.md [hunk 7]
```diff
@@ -29,6 +29,35 @@ There are three types of log files automatically produced by the client, which c
...
+
+Windows
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:42]:
```suggestion
Windows:

```

File: Teams/log-files.md [hunk 8]
```diff
@@ -29,6 +29,35 @@ There are three types of log files automatically produced by the client, which c
...
+Crtl + Alt + Shift + 1
+Mac
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:44]:
```suggestion

Mac:

```

File: Teams/log-files.md [hunk 9]
```diff
@@ -29,6 +29,35 @@ There are three types of log files automatically produced by the client, which c
...
+Option + Command + Shift + 1
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:45]:
```suggestion
Option + Command + Shift + 1

```

File: Teams/log-files.md [hunk 10]
```diff
@@ -29,6 +29,35 @@ There are three types of log files automatically produced by the client, which c
...
+Media logging is turned off by default. To enable Media logging, users must turn on the option in the Teams client. Go to Settings > General, select the Enable logging for meeting diagnostics (requires restarting Teams). The Teams client must be restarted for logging to begin.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:46]:
```suggestion
Media logging is turned off by default. To enable Media logging, users must turn on the option in the Teams client. Go to **Settings** > **General**, and select **Enable logging for meeting diagnostics (requires restarting Teams)**. The Teams client must be restarted for logging to begin.

```

File: Teams/log-files.md [hunk 11]
```diff
@@ -29,6 +29,35 @@ There are three types of log files automatically produced by the client, which c
...
+Note If Media logging is enabled there will be additional files included in the Meeting folder which are necessary for investigating audio and video issues. If Media logging is not enabled there will be a limited number of logs available.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:47]:
```suggestion
> [!NOTE]
> If Media logging is enabled, there will be additional files included in the Meeting folder which are necessary for investigating audio and video issues. If Media logging is not enabled, there will be a limited number of logs available.
```

File: Teams/log-files.md [hunk 12]
```diff
@@ -29,6 +29,35 @@ There are three types of log files automatically produced by the client, which c
...
+Note In this article, the term Debug logs refers to the logs that are used for troubleshooting. However, the files that are generated for these logs will contain the term diagnostic logs in their names.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:48]:
```suggestion
> [!NOTE] In this article, the term _Debug logs_ refers to the logs that are used for troubleshooting. However, the files that are generated for these logs will contain the term _Diagnostic logs_ in their names.

```

File: Teams/log-files.md [hunk 13]
```diff
@@ -29,6 +29,35 @@ There are three types of log files automatically produced by the client, which c
...
+The following table outlines the various clients and their associated logs. 
+TABLE 1
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:50]:
```suggestion
```

File: Teams/log-files.md [hunk 14]
```diff
@@ -29,6 +29,35 @@ There are three types of log files automatically produced by the client, which c
...
+Client	Debug	Desktop	Media
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:51]:
```suggestion

| Client | Debug | Desktop | Media |

```

File: Teams/log-files.md [hunk 15]
```diff
@@ -29,6 +29,35 @@ There are three types of log files automatically produced by the client, which c
...
+Web	X	-	-
+Windows	X	X	X
+Mac OSX	X	X	X
+Linux	X	X	X
+iOS	-	-	-
+Android	-	-	-
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:57]:
```suggestion
```

File: Teams/log-files.md (hunk 12 above)
Comment (JohanFreelancer9) [reply Teams/log-files.md:48]: Duplication.

File: Teams/log-files.md [hunk 16]
```diff
@@ -29,6 +29,35 @@ There are three types of log files automatically produced by the client, which c
...
+For a complete list of supported operating systems and browsers, see Get clients for Microsoft Teams.
+
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:59]:
```suggestion
```

Comment (JohanFreelancer9) [reply Teams/log-files.md:59]: Duplication

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - I'll do another review once my suggested changes have been committed.

**Comment by scanum** [comment]
@anurag1128 Please commit suggested changes.

**Pull request closed without merging**